### PR TITLE
Feature/add patch db user mapper tests

### PR DESCRIPTION
**Pull request by TIove**

### CODE REVIEW

File: src/UserService.Mappers/Models/PatchDbUserMapper.cs [hunk 1]
```diff
@@ -16,35 +17,6 @@ public class PatchDbUserMapper : IPatchDbUserMapper
     {
         private readonly ILogger<PatchDbUserMapper> _logger;
         private readonly IRequestClient<IAddImageRequest> _requestClient;
```
Comment (djino97) [inline src/UserService.Mappers/Models/PatchDbUserMapper.cs:19]: remove, not used

File: src/UserService.Mappers/Models/PatchDbUserMapper.cs [hunk 2]
```diff
@@ -54,7 +26,9 @@ public class PatchDbUserMapper : IPatchDbUserMapper
             _requestClient = requestClient;
```
Comment (djino97) [inline src/UserService.Mappers/Models/PatchDbUserMapper.cs:26]: remove logger and requestClient

File: test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    {
+        private Mock<ILogger<PatchDbUserMapper>> _loggerMock;
+        private Mock<IRequestClient<IAddImageRequest>> _rcImageMock;
+        private IPatchDbUserMapper _mapper;
+
+        private JsonPatchDocument<EditUserRequest> _request;
+        private JsonPatchDocument<DbUser> _response;
+        
+        
+
```
Comment (MariyaSin) [inline test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs:34]: a lot of empty lines...

File: test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                    $"/{nameof(DbUser.Status)}",
+                    "",
+                    UserStatus.Vacation)
+            }, new CamelCasePropertyNamesContractResolver());
+        }
+        
+        [SetUp]
+        public void SetUp()
+        {
+        }
```
Comment (MariyaSin) [inline test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs:94]: why this test if it is empty?

File: src/UserService.Business/Commands/User/EditUserCommand.cs [hunk 5]
```diff
@@ -109,7 +109,7 @@ public OperationResultResponse<bool> Execute(Guid userId, JsonPatchDocument<Edit
             
             GetAvatarImageId(avatarRequest?.value as string, errors);
```
Comment (djino97) [inline src/UserService.Business/Commands/User/EditUserCommand.cs:110]: 107 - 110 lines need remove

Comment (TIove) [reply src/UserService.Business/Commands/User/EditUserCommand.cs:110]: fix

File: test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs (hunk 4 above)
Comment (TIove) [reply test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs:94]: fix, I have OneTimeSetUp

File: test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+        private JsonPatchDocument<EditUserRequest> _request;
+        private JsonPatchDocument<DbUser> _response;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            _mapper = new PatchDbUserMapper();
+
+            _request = new JsonPatchDocument<EditUserRequest>(new List<Operation<EditUserRequest>>
```
Comment (djino97) [inline test/UserService.Mappers.UnitTests/PatchDbUserMapperTests.cs:29]: rename _request  => _result

**Review by djino97 — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Review by 24eeeee — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**